content/projects/gsoc/2020/projects/git-plugin-performance.adoc

```diff
@@ -23,44 +24,113 @@ links:
+
+. Design a strategy to implement performance improvements in the plugin
+
+==== Resources
+
+. Pull Requests:
+  - link:https://github.com/jenkinsci/git-client-plugin/pull/553[Adding JMH benchmarks module]
+  - link:https://github.com/jenkinsci/git-client-plugin/pull/556[Optimising benchmarks module]
+  - link:https://github.com/jenkinsci/git-plugin/pull/904[Fixing JENKINS-49757]
+  - link:https://github.com/jenkinsci/git-plugin/pull/927[Adding a safety switch to retain second fetch]
+. link:/blog/2020/07/09/git-performance-improvement-phase1/[Coding phase 1 summary blog]
+. link:https://docs.google.com/presentation/d/1kaSuYWLBnSvWJWq4lkH5ecOdN-m8n-uymdYHVXcv-EU/edit?usp=sharing[Slides]
+. Phase 1 demo and presentation:
+
+video::HQLhakpx5mk[youtube,start=1791,width=800,height=420]
+
+=== Evaluation Phase 2
+
+. Implementing performance improvement from benchmarks:
+  - GitToolChooser - A class which tries to recommend the optimal git implementation for a particular remote repository
+  - link:https://github.com/jenkinsci/git-plugin/blob/master/src/main/java/jenkins/plugins/git/GitToolChooser.java[Repository]
+
+. Widening the scope of benchmarking in the git client plugin
+  - Benchmarks on multiple repository parameters
+  - Benchmarks on multiple platforms
+
+==== Resources
+
+. Pull Requests:
+  - link:https://github.com/jenkinsci/git-plugin/pull/931[Adding GitToolChooser]
+. link:/blog/2020/07/29/git-performance-improvement-phase2/[Coding phase 2 summary blog with benchmark results]
+. link:https://docs.google.com/presentation/d/1aIr1aOZSE59KeQ8-Hr_mbOYMctZLUlCELJNVuzWAcz4/edit?usp=sharing[Slides]
+. Phase 2 demo and presentation: 
+
+video::b67I6spBdTg[youtube,start=311,width=800,height=420]
+
+=== Evaluation Phase 3
+
+This phase marked the release of the git plugin with all of the performance improvements performed during GSoC period. That includes addition of GitToolChooser, removing redundant second fetch from checkout and adding a JMH benchmark module in git client plugin.
+
+. Git Plugin 4.4.0 released
+  - link:https://github.com/jenkinsci/git-plugin/releases/tag/git-4.4.0[Changelog]
+  - link:https://github.com/jenkinsci/git-plugin/blob/git-4.4.0/README.adoc#global-configuration[Documentation]
+
+. Git Client Plugin 3.4.0 released
+  - link:https://github.com/jenkinsci/git-client-plugin/releases/tag/git-client-3.4.0[Changelog]
+  - link:https://github.com/jenkinsci/git-client-plugin/blob/git-client-3.4.0/README.adoc[Documentation]
+
+==== Resources
+
+. Pull Requests:
+  - link:https://github.com/jenkinsci/git-plugin/pull/931[GitToolChooser]
+  - link:https://github.com/jenkinsci/git-client-plugin/pull/594[Add UnsupportedCommand to the git client plugin]
+  - link:https://github.com/jenkinsci/git-client-plugin/pull/601[Bug fix related to UnsupportedCommand]
+. Blog: TBA
+. Slides: https://docs.google.com/presentation/d/1rAjjF_pBjtGDaaC8rgReqVzd9WJ4Z0jZTvaPHabk3SI/edit?usp=sharing
```

Comment:
Since the other long links have been converted, would be nice to convert this one as well
```suggestion
. link:https://docs.google.com/presentation/d/1rAjjF_pBjtGDaaC8rgReqVzd9WJ4Z0jZTvaPHabk3SI/edit?usp=sharing[Slides]
```